### PR TITLE
fix(docs): Convert LinkTo to regular links

### PR DESCRIPTION
**Pull request by imagoiq**

### CODE REVIEW

**Comment by imagoiq** [comment]
> We have a lot of links included in the stories files, which are broken. Would be nice to fix them as well in this PR.
> 
> Example: alert.stories.ts -> meta.argTypes.icon.description
> 
> An even better idea could be to add a e2e test in documentation, which goes trough every a-tag and checks if the url in the href can be loaded. What do you think, should we open a ticket for this?

For the record, I opened this ticket https://github.com/swisspost/design-system/issues/2465

I fixed the broken links in meta (at least the one that I've found).

**Comment by oliverschuerch** [comment]
Please update to the newest `main` branch, as the documentation isn't running correctly.

**Review by oliverschuerch — CHANGES_REQUESTED** [review]
Please update to the newest main branch, as the documentation isn't running correctly.

File: packages/documentation/src/stories/components/forms/choice-card/choice-card.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const choiceCardMeta: Meta = {
     disabled: {
       name: 'Disabled',
       description:
-        'When set to `true`, disables the component\'s functionality and places it in a disabled state.<span className="mt-mini alert alert-info alert-sm">There are accessibility concerns with the disabled state.<br/>Please read our <a href="/?path=/docs/foundations-accessibility--page#disabled-state">disabled state accessibility guide</a>.</span>',
+        'When set to `true`, disables the component\'s functionality and places it in a disabled state.<span className="mt-mini alert alert-info alert-sm">There are accessibility concerns with the disabled state.<br/>Please read our <a href="/?path=/docs/foundations-accessibility--docs#disabled-state">disabled state accessibility guide</a>.</span>',
```
Comment (imagoiq) [inline packages/documentation/src/stories/components/forms/choice-card/choice-card.ts:63]: In those controls, html seem to no work anymore. I don't get it. @oliverschuerch do you have an idea?

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**